scripts/image_converter.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+
+check_dependencies() {
+    # Check if ImageMagick is installed
+    if ! command -v convert &> /dev/null; then
+        echo "Error: ImageMagick is not installed. Please install it."
+        exit 1
+    fi
+}
+
+check_parameters() {
+    # Check if all mandatory parameters are provided
+    if [[ -z "$FROM" || -z "$TO" || -z "$IN" ]]; then
+        echo "Error: Missing mandatory parameters. Please provide --from, --to, and --in."
+        echo "$FROM"
+        echo "$TO"
+        echo "$IN"
```

Comment:
Are these `echo` intentional? If so, I suggest changing them to something like:

```bash
echo "From: $FROM"
echo "To: $TO"
echo "In: $IN
```
Check it out:

![image](https://github.com/laradru/ai-palatal-rugoscopy/assets/26142253/53338177-c51b-42fa-be95-ef73fee6e6c0)

Reply:
Oh, I shouldn't have checked these lines in.